docs/topics/extensions.rst

```diff
@@ -243,6 +243,30 @@ An extension for debugging memory usage. It collects information about:
 To enable this extension, turn on the :setting:`MEMDEBUG_ENABLED` setting. The
 info will be stored in the stats.
 
+Spider state extension
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. module:: scrapy.extensions.spiderstate
+   :synopsis: Spider state extension
+
+.. class:: SpiderState
+
+Manages spider state data by loading it before a crawl and saving it after.
+
+Give a value to the :setting:`JOBDIR` setting to enable this extension.
+When enabled, this extension manages the :attr:`~scrapy.Spider.state` 
+attribute of your :class:`~scrapy.Spider` instance:
+    
+-   When your spider closes (:signal:`spider_closed`), the contents of its 
+    :attr:`~scrapy.Spider.state` attribute are serialized into a file named 
+    ``spider.state`` in the :setting:`JOBDIR` folder.
+-   When your spider opens (:signal:`spider_opened`), if a previously-generated 
+    ``spider.state`` file exists in the :setting:`JOBDIR` folder, it is loaded 
+    into the :attr:`~scrapy.Spider.state` attribute.
+
+
+For an example, see :ref:`topics-keeping-persistent-state-between-batches`.
```

Comment:
The text linked here says "a built-in extension", can you please make that a link to this text (by linking to `SpiderState`)?